src/api.h

```diff
@@ -402,7 +402,7 @@ return __p.invoke(func);\
 
 #endif
 
-
+    #define VALIDATE_FIXED_SIZE(ARG, SIZE) if((ARG) != (SIZE)) { std::ostringstream ss; ss << "Unsupported FW image size provided { " << ARG << " }" " expecting { " << SIZE << " }"; throw librealsense::invalid_value_exception(ss.str()); }
```

Comment:
The Macro's name is generic one, therefore the generated error message should not refer to FW Image